resource/writers/match_writers.hpp

```diff
@@ -108,6 +129,7 @@ class jgf_match_writers_t : public match_writers_t
 
     json_t *m_vout = NULL;
     json_t *m_eout = NULL;
+    std::string nodename;
```

Comment:
Fluxion designates member variables with `m_`:
```suggestion
    std::string m_nodename;
```